rlclientlib/logger/logger_facade.cc

```diff
@@ -161,6 +161,20 @@ namespace reinforcement_learning {
       }
     }
 
+    int observation_logger_facade::log(const char* event_id, const char* index, float outcome, api_status* status) {
```

Comment:
nit: following this down to here the arguments have been passed as: `outcome, index`, can we keep that here too instead of switching to `index, outcome`?

Reply:
good catch. fixed the order and naming to be consistent everywhere.